pyproject.toml

```diff
@@ -13,7 +13,7 @@ license = {text = "Apache 2"}
 dependencies = [
     "fms-model-optimizer[fp8]>=0.6.0",
     "ibm-fms>=1.2.1",
-    "vllm>=0.9.2,<=0.10.0",
+    "vllm>=0.9.2,<=0.10.1",
```

Comment:
do we still need to support 0.9.2?

Reply:
Not sure... we only bumped the patch version and not the minor one...